accounts/pkg/proto/v0/accounts.pb.micro_test.go

```diff
@@ -617,7 +617,7 @@ func TestUpdateNonUpdatableFieldsInAccount(t *testing.T) {
 				errMsg := fmt.Sprintf("can not update field %s, either unknown or readonly", tt.updateMask[0])
 				assert.Equal(t, errMsg, e.Detail)
 			} else {
-				t.Fatal("Expected merror errors but found something else.")
+				t.Fatal("Expected merrors errors but found something else.")
```

Comment:
This appears to be the name of the library...

Reply:
Yes, but your fix ist still valid since we're using it in plural throughout the codebase